pom.xml

```diff
@@ -118,7 +118,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <version.spring.boot>2.7.7</version.spring.boot>
         <version.guava>31.1-jre</version.guava>
-        <version.spring.cloud>2021.0.5</version.spring.cloud>
+        <version.spring.cloud>2022.0.1</version.spring.cloud>
```

Comment:
This is for `spring-cloud-azure-4.x` profile.

Reply:
`spring-cloud-azure-5.x` is using `2022.0.0` now. I'll create a PR manually to upgrade to `2022.0.1`.

Reply:
PR created: https://github.com/Azure-Samples/azure-spring-boot-samples/pull/650